test/test_traverse.py

```diff
@@ -39,3 +40,8 @@ def test_a(self):
                 ('RawText', 'Emphasis', 4),
             ]
         )
+
+    def test_b(self):
```

Comment:
A small suggestion:

```suggestion
    def test_with_class_filter(self):
```

Also, to have more meaningful testing methods names, we could also rename the first testing method, e.g. to `test_with_included_source`?